CHANGELOG.md

```diff
@@ -1,5 +1,98 @@
 ## Changelog
 
+### 0.14.0 [01-04-2020]
+
+#### General
+
+* Improved key validation: require to have at least one valid, non-expiring self signature.
+* Added support for 'stripped' keys without userids and certifications but with valid subkey binding signature.
+* Added support for Windows via MinGW/MSYS2.
+* Added support for Windows via MSVC.
+* Fixed secret key locking when it is updated with new signatures/subkeys.
+* Refactored CLI internals to use the single key search function.
+* Removed `ssize_t` usage for better cross-platform support.
```

Comment:
I think it is commonly accepted that changelog should contain changes that matter to users, i.e. new features, bugs fixed, changes in public API. Internal code changes don't belong here, developers can look them in Git history if they are interested.